packages/angular/projects/ui-angular/tsconfig.spec.json

```diff
@@ -1,10 +1,3 @@
-/* To learn more about this file see: https://angular.io/config/tsconfig. */
 {
-  "extends": "../../tsconfig.json",
-  "compilerOptions": {
-    "outDir": "../../out-tsc/spec",
-    "types": ["jasmine"]
-  },
-  "files": ["src/test.ts"],
-  "include": ["**/*.spec.ts", "**/*.d.ts"]
+  "extends": "../../tsconfig.json"
```

Comment:
If all this file does is extend the base `tsconfig`, do we need it all?

Reply:
I'll remove it, somehow it was there twice.

Reply:
b226db3